app/bingey-api/package.json

```diff
@@ -19,7 +20,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon server.js",
-    "postinstall": "yarn basetag link"
+    "postinstall": "yarn basetag link",
+    "import-data": "node --max_old_space_size=2048 utils/reader.js"
```

Comment:
The terminal that is running this command seems to need more memory, I left the script spinning when I was away, 12k movies were uploaded but then I got a heap memory exception, It's pretty difficult to test this so I'll try again with expanded memory